EnglishFilter/sections/whitelist.txt

```diff
@@ -139,6 +139,9 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/38089
 @@||infowars.com^$csp=script-src 'self' *.leadpages.net *.gstatic.com *.google.com *.googleapis.com *.playwire.com *.facebook.com *.bootstrapcdn.com *.twitter.com *.spot.im
 @@||api.infowarsmedia.com/videojs-event-tracking/dist/videojs-event-tracking.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/34162
```

Comment:
Please move rules to Mobile filter.

Reply:
I moved whitelist rule to Mobile Filter.